Routers/BookingRouter.cs

```diff
@@ -0,0 +1,118 @@
+﻿using AirportTicketBooking.Classes;
+using AirportTicketBooking.Repositories;
+using ClassType = TicketClassType.ClassType;
+
+namespace AirportTicketBooking.Routers
+{
+    internal class BookingRouter
+    {
+        private BookingRepositoty _bookingRepositoty;
+        private FlightRouter _flightRouter;
+
+        public BookingRouter()
+        {
+            _bookingRepositoty = new BookingRepositoty();
+        }
+
+        public List<Booking> GetAllData()
+        {
+            return _bookingRepositoty.Load();
+        }
+
+        public List<Booking> GetAll()
+        {
+            return GetAllData().Where(booking => booking.ValidationMessage == null).ToList();
+        }
+
+        public bool TryGetByBookingId(int bookingId, out Booking booking)
+        {
+            booking = GetAll().FirstOrDefault(booking => booking.BookingId == bookingId);
+            return (booking != null);
+        }
+
+        public void Add(Booking booking)
+        {
+            _bookingRepositoty.Save(booking);
+        }
+
+        public void RemoveByBookingId(int bookingId)
+        {
+            if (!TryGetByBookingId(bookingId, out _))
+            {
+                throw new Exception($"Booking with BookingId: {bookingId} doesn't exist");
+            }
+
+            var remainingBookings = GetAll().Where(booking => booking.BookingId != bookingId).ToList();
+
+            _bookingRepositoty.DeleteTheFileData();
+
+            foreach (var booking in remainingBookings)
+            {
+                _bookingRepositoty.Save(booking);
+            }
+        }
+
+        public List<Booking> GetBookingForPassenger(int passengerPassportId)
```

Comment:
NIT: public List<Booking> GetBooking(int passengerPassportId) instead of the current one. 
You can know from the signature that this will return the booking for that passenger since we are using the passengerPassportId. 

The same thing for the below methods.